packs/actions/opportune-riposte.json

```diff
@@ -18,12 +18,25 @@
             "remaster": false,
             "title": "Pathfinder Advanced Player's Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "opportune-riposte",
+                "predicate": [
+                    {
+                        "or": [
+                            "feat:felicitous-riposte",
+                            "feat:impossible-riposte",
+                            "feature:eternal-confidence"
+                        ]
+                    }
```

Comment:
Not sold on this idea myself, but: Instead of predicating on these specific feats we can predicate on a more generic roll option, something like `add-opportune-riposte-toggle`, then set that option with those feats. This would make it expandable by 3pp or if there's any changes in PC2 to what needs this it's easy to do.